pkg/api/platformapi/stackapi/delete.go

```diff
@@ -18,27 +18,30 @@
 package stackapi
 
 import (
+	"context"
 	"errors"
-	"io"
 	"strings"
 
 	"github.com/blang/semver"
 
 	"github.com/elastic/cloud-sdk-go/pkg/api"
 	"github.com/elastic/cloud-sdk-go/pkg/api/apierror"
+	"github.com/elastic/cloud-sdk-go/pkg/client/stack"
 	"github.com/elastic/cloud-sdk-go/pkg/multierror"
+	"github.com/elastic/cloud-sdk-go/pkg/util/ec"
 )
 
-// GetParams is consumed by Get
-type GetParams struct {
+// DeleteParams is consumed by Delete
+type DeleteParams struct {
 	*api.API
+	Region  string
 	Version string
 }
 
 // Validate ensures that the parameters are usable by the consuming
 // function
-func (params GetParams) Validate() error {
-	var merr = multierror.NewPrefixed("stack get")
+func (params DeleteParams) Validate() error {
+	var merr = multierror.NewPrefixed("stack delete")
```

Comment:
I've been prefixing these errors as `invalid stack delete params` rather than `stack delete`. I think it makes the error more decipherable by the user, right?

Reply:
sounds good!